### DISCUSSION TITLE
Hugging face embedding model `intfloat/e5-small-v2` (Sentence Transformer) to IR with NNCF optimised

### DISCUSSION THREAD

**Discussion by AmanRavan** [openvinotoolkit/openvino#23853] [Q&A]
I am trying some hands on OPENVINO.

Not able to convert `intfloat/e5-small-v2` from Hugging face to IR with NNCF optimised for faster execution.

I also took reference from  [Example From OpenVino](https://docs.openvino.ai/2024/notebooks/254-rag-chatbot-with-output.html)

or any other way how I can convert my embedding model to IR optimised with NNCF using openvino.

**Comment by RitikaxShakya** [comment]
Hello @AmanRavan ! Did you try https://github.com/openvinotoolkit/openvino.genai/tree/master/text_generation/causal_lm/cpp this first? Maybe first try this example model "TinyLlama-1.1B-Chat-v1.0" with this reference.

**Comment by AmanRavan** [comment]
Thanks, 

Yup, that's my point.
I am able to use LLM in openvino but not the Embedding model(text-embedding-ada-xxx or some open source).

Meanwhile..Just diving deeper

I was going through this [#254llmragopenvino](https://github.dev/openvinotoolkit/openvino_notebooks/blob/main/notebooks/254-llm-chatbot/254-rag-chatbot.ipynb)

&&&

there was one line ``from ov_embedding_model import OVEmbeddings`` and one more thing what happened to embedding model. 

Any idea @RitikaxShakya, suggestions on embedding model running in openvino.
